web/pdf_page_view.js

```diff
@@ -513,6 +513,20 @@ class PDFPageView {
     }
   }
 
+  /**
+   * @typedef {Object} PDFPageViewUpdateParameters
+   * @property {number} [scale] The new scale, if specified.
+   * @property {number} [rotation] The new rotation, if specified.
+   * @property {Promise<OptionalContentConfig>} [optionalContentConfigPromise]
+   *   A promise that is resolved with an {@link OptionalContentConfig} instance.
+   *   The default value is `null`.
```

Comment:
The following should fix the linting error:
```suggestion
   *   A promise that is resolved with an {@link OptionalContentConfig}
   *   instance. The default value is `null`.
```